src/cpp/desktop/DesktopSessionLauncher.cpp

```diff
@@ -555,15 +557,73 @@ void SessionLauncher::onLaunchFirstSession()
    }
 }
 
+namespace {
+
+#ifdef _WIN32
+
+FilePath resolveSessionPathWin32(const FilePath& defaultPath)
```

Comment:
We'll need to port this over to Electron as well (in `main`). Perhaps open an issue to track doing that?

Reply:
Will do!